### PR TITLE
Fixed the app name

### PR DESCRIPTION
**Pull request by suyashbansal**
Updated the app name to The Bollywood Song Quiz

### CODE REVIEW

File: functions/index.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ const TTS_DELAY = '500ms';
 const MAX_PREVIOUS_QUESTIONS = 100;
 const SUGGESTION_CHIPS_MAX_TEXT_LENGTH = 25;
 const SUGGESTION_CHIPS_MAX = 8;
-const GAME_TITLE = 'The Bollywod Song Quiz';
+const GAME_TITLE = 'The Bollywood Song Quiz';
```
Comment (aayushibansal) [inline functions/index.js:98]: Why did we change this?

**Review by aayushibansal — COMMENTED** [review]
Please incorporate the review comments.

File: functions/index.js (hunk 1 above)
Comment (suyashbansal) [reply functions/index.js:98]: The spelling was incorrect initially.

**Review by aayushibansal — APPROVED** [review]
LGTM

**Pull request merged**